attributes/default.rb

```diff
@@ -95,6 +95,7 @@
       '/media/downloadable' => 'restricted',
       '/pkginfo' => 'restricted',
       '/report/config.xml' => 'restricted',
+      '^/rss/(catalog/(notifystock|review)|order/new)(/|$)' => 'restricted',
```

Comment:
that seems unlikely. Definitely not supported in nginx, and still would be weird in apache. I'll check tho. I've seemed to have missed off (/index\.php)? there tho